extensions/shared/components/audio-player/index.jsx

```diff
@@ -72,8 +74,15 @@ function AudioPlayer( {
 
 	useEffect( () => {
 		const audio = audioRef.current;
-		// Initialize MediaElement.js.
-		const mediaElement = new MediaElementPlayer( audio, meJsSettings );
+		const mediaElement = new MediaElementPlayer( audio, {
+			...meJsSettings,
+			success: function() {
+				dispatch( STORE_ID ).registerMediaSource( audio.id, {
+					status: 'is-paused',
+					timestamp: 0,
+				} );
+			}
+		} );
```

Comment:
Does this mean we're only taking the initial, default tracks into account? If I extend the number of tracks later on via the block settings, should a new media source be added to the source tree for each one of the new tracks?

Reply:
It's a good question. From my point of view, should exist a media-source register per audio player. Right now, when you add a podcast player, it inserts only one `<audio>` element. The block has the ability to switch among different sources (tracks), but the audio instance is always the same.
So the answer is not, it should not register a new media-source for each track but for the podcast player.

Something to keep in mind: This is just the initial step of the global state implementation. We already have follow-up PRs with significant changes. However, this PR provides the initial structure.